### PR TITLE
v1.82.0 - Component updates.

### PR DESCRIPTION
**Pull request by DamianMullins**
### Added
- `c-badge` component small rounded modifier styles.
- `u-spacingBottom--large--aboveMid` & `u-shadowBottom--belowMid` helper to the utilities.

### Changed
- Updated `c-card` component highlight modifier styles.
- Updated `c-modal` component styles.
- Consistent spacing between rules in the utilities module.

### Removed
- `c-card` component large rounded modifier styles.

### CODE REVIEW

File: src/scss/components/_cards.scss [hunk 1]
```diff
@@ -12,8 +12,7 @@ $card-bgColor--disabled                         : $grey--lightest;
 $card-tooltip-width                             : 10px;
 $card-arrow-bottom-position                     : 0;
 $card-padding                                   : spacing(x2);
-$card-radius                                    : 4px;
-$card-radius--large                             : 8px;
+$card-radius                                    : spacing(x0.5);
```
Comment (ashleynolan) [inline src/scss/components/_cards.scss]: Not a big deal as it's unlikely to change atm, but not sure we'd want spacing linked to the border-radius like this. Would be weird if the spacing increased that corners of components also were affected.

Comment (DamianMullins) [reply src/scss/components/_cards.scss]: Yeah, good shout, I get a bit carried away with the spacing helper sometimes! I'll update this soon when I next put a PR in.

Comment (DamianMullins) [reply src/scss/components/_cards.scss]: This was done in #280.

Comment (DamianMullins) [reply src/scss/components/_cards.scss]: Ah no it wasn't, I updated the wrong thing!

Comment (DamianMullins) [reply src/scss/components/_cards.scss]: Fixed in #281 😸

**Pull request merged**